### PR TITLE
Feature/issue 64/furnish database

### PR DESCRIPTION
**Pull request by QahtanAlJammali**
# [Issue #64 ] https://github.com/gcivil-nyu-org/wed-fall24-team5/issues/64

## Description

Created the database app, and furnished it with the models.py that would create the database tables, and tested it out with pgAdmin with SQL queries, we are ready to sunset SQLite.

## Change Type (delete non-relevant options)

- 💡 New feature (non-breaking change which adds functionality)
- 🌐 Release (for preparing a release branch)
- 🧪 Test (for adding tests)

## Dependencies

-N/A

### CODE REVIEW

File: src/accounts/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+                        editable=False,
+                        primary_key=True,
+                        serialize=False,
+                    ),
+                ),
+                ("organization_name", models.CharField(max_length=255)),
+                (
+                    "type",
+                    models.CharField(
+                        choices=[
```
Comment (arsalananwar11) [inline src/accounts/migrations/0001_initial.py:31]: Can we also have self and others?

**Review by arsalananwar11 — CHANGES_REQUESTED** [review]

File: src/accounts/migrations/0001_initial.py (hunk 1 above)
Comment (QahtanAlJammali) [reply src/accounts/migrations/0001_initial.py:31]: Just did in my latest commit

**Review by maxgrove42 — COMMENTED** [review]
nice work!!

**Review by maxgrove42 — APPROVED** [review]
meant to hit approve

**Comment by QahtanAlJammali** [comment]
@arsalananwar11 approve your requested changes please!

**Review by TekuriSaiAkhil — APPROVED** [review]
Looks good, Great work @QahtanAlJammali

**Review by estchang — APPROVED** [review]
Works great, thank you!

**Review by arsalananwar11 — APPROVED** [review]
Looks great! Thanks @QahtanAlJammali

**Pull request merged**